src/base/settingsstorage.h

```diff
@@ -84,6 +86,11 @@ class SettingsStorage final : public QObject
             const typename T::Int value = loadValue(key, static_cast<typename T::Int>(defaultValue));
             return T {value};
         }
+        else if constexpr (std::same_as<T, std::chrono::seconds>)
```

Comment:
IMO, it's redundant. I would just use `CachedSettingValue<qint64>` than add all these additional types `std::chrono::seconds`, `std::chrono::milliseconds` etc. here.

Reply:
I think its better to keep it that way, so it can be reused in other places that reference time because i think it might be safer to keep the types as time types and keep the conversion abstract for the developer.

I might suggest even pushing more into using chrono types, for example changing CachedSettingValue<int> for other intervals into chrono types:

example:
change m_storeRefreshInterval type to min
```cpp
        CachedSettingValue<int> m_storeRefreshInterval;
        CachedSettingValue<std::chrono::seconds> m_storeFetchDelay;
```
into
```cpp
        CachedSettingValue<std::chrono::minutes> m_storeRefreshInterval;
        CachedSettingValue<std::chrono::seconds> m_storeFetchDelay;
```

let me know if you accept or reject this so i know if i should make another update to this PR.

Reply:
> I might suggest even pushing more into using chrono types, for example changing CachedSettingValue for other intervals into chrono types

At least not in this PR.

Reply:
I fixed as requested at https://github.com/qbittorrent/qBittorrent/pull/19801/commits/ea4ad45c85b8284efdb076e06976d05bbcda111b , I might draft a separate PR for that in the future